astropy/coordinates/tests/test_solar_system.py

```diff
@@ -376,21 +381,21 @@ def test_earth_barycentric_velocity_multi_d():
 @pytest.mark.remote_data
 @pytest.mark.skipif('not HAS_JPLEPHEM')
 @pytest.mark.parametrize(('body', 'pos_tol', 'vel_tol'),
-                         (('mercury', 1000.*u.km, 1.*u.km/u.s),
+                         (('mercury', 1500.*u.km, 1.*u.km/u.s),
```

Comment:
I'm not sure I understand why you need this increased tolerance? Using current astropy master....

```python
from astropy.coordinates import get_body_barycentric_posvel
from astropy.time import Time

body = 'mercury'
t = Time('2016-03-20T12:30:00')
ep, ev = get_body_barycentric_posvel(body, t, ephemeris='builtin')
dp, dv = get_body_barycentric_posvel(body, t, ephemeris='de432s')
dp2, dv2 = get_body_barycentric_posvel(body, t, ephemeris='http://naif.jpl.nasa.gov/pub/naif/generic_kernels/spk/planets/de440s.bsp')
```

```python
In [34]: (dp-ep).norm()
Out[34]: <Quantity 200.40124694 km>

In [35]: (dp2-ep).norm()
Out[35]: <Quantity 135.14350477 km>
```

Both seem well within the 1000 km tolerance specified. Earth does seem to need the 120km tolerance though but this is due to a genuine difference in position between de440 and de4332, so I guess that's OK!